admin_guide/topics/testing-auto-scaler.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// * admin_guide/cluster-autoscaler.adoc
+
+[id='testing-AWS-cluster-auto-scaler_{context}']
+= Testing the auto-scaler
+
+After you install or upgrade your cluster that uses the auto-scaler, you can
+confirm that auto scaling works.
+
+.Prerequisites
+
+* Your {product-title} cluster on Amazon Web Services (AWS) uses the
+auto-scaler.
+
+
+. Create the *_scale-up.yaml_* file that contains the deployment configuration
+to test auto-scaling:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: scale-up
+  labels:
+    app: scale-up
+spec:
+  replicas: 20
+  selector:
+    matchLabels:
+      app: scale-up
+  template:
+    metadata:
+      labels:
+        app: scale-up
+    spec:
+      containers:
+      - name: busybox
+        image: docker.io/library/busybox
+        resources:
+          requests:
+            memory: 2Gi
+        command:
+        - /bin/sh
+        - "-c"
+        - "echo 'this should be in the logs' && sleep 86400"
+      terminationGracePeriodSeconds: 0
+----
++
+This deployment specifies 20 replicas, but the initial size of the cluster
+cannot run all of the pods without first increasing the number of compute nodes.
+
+. Deploy the configuration:
++
+[source,bash]
+----
+$ oc apply -n autoscaler-demo -f scale-up.yaml
+----
++
+[NOTE]
+====
+The deployment must run in the same namespace that the cluster-autoscaler
```

Comment:
why?

Reply:
Yes, this is wrong. I will update the examples to add and propagate the following label on the ASG:

```yaml
k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute: 'true'
```



k8s.io/cluster-autoscaler/node-template/label/node-role.kubernetes.io/compute |  
-- | --